src/neural/cuda/fp16_kernels.cu

```diff
@@ -443,7 +443,7 @@ void OutputInputTransform(int N, int C, int se_K, T* output, const T* input,
           72 * C * sizeof(half));
       OutputInputTransformKernel_fp16_shmem_board<activation, use_bias,
                                                   use_skip>
-          <<<N, C, kMaxResBlockFusingSeFp16AmpereSmem, stream>>>(
+          <<<N, C, 72 * C * sizeof(half), stream>>>(
```

Comment:
Stray diff?

Reply:
No, it's actually a fix that I forgot to include with my last change (otherwise nets bigger than 384 filters and smaller than 512 filters would fail here).